### PR TITLE
[feat] Create Storybook for ui kit v2

### PR DESCRIPTION
**Pull request by arsengit**
Initialize Storybook for new Aim UI kit

### CODE REVIEW

**Comment by arsengit** [comment]
Resolves #2249

File: aim/web/ui/.storybook/main.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+    "../src/**/*.stories.@(js|jsx|ts|tsx)"
+  ],
+  "addons": [
+    "@storybook/addon-links",
+    "@storybook/addon-essentials",
+    "@storybook/addon-interactions",
+    "@storybook/preset-create-react-app"
+  ],
+  "framework": "@storybook/react"
+}
```
Comment (KaroMourad) [inline aim/web/ui/.storybook/main.js:13]: let's add new line at the end of file to fix linting issue

File: aim/web/ui/.storybook/preview.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+export const parameters = {
+  actions: { argTypesRegex: "^on[A-Z].*" },
+  controls: {
+    matchers: {
+      color: /(background|color)$/i,
+      date: /Date$/,
+    },
+  },
+}
```
Comment (KaroMourad) [inline aim/web/ui/.storybook/preview.js:9]: let's add new line at the end of file to fix linting issue

**Review by roubkar — APPROVED** [review]

**Review by VkoHov — APPROVED** [review]
lgtm!

**Pull request merged**